setup/rt_literal_property_attrs_doc.json

```diff
@@ -37,6 +37,48 @@
         }
       ]
     },
+    {
+      "@id": "_:b4",
+      "@type": [
+        "http://sinopia.io/vocabulary/PropertyTemplate"
+      ],
+      "http://www!w3!org/2000/01/rdf-schema#label": [
+        {
+          "@value": "Validation Data Type"
+        }
+      ],
+      "http://sinopia!io/vocabulary/hasRemark": [
+        {
+          "@value": "Data Type to validate the literal, e.g. integer or dateTime."
+        }
+      ],
+      "http://sinopia!io/vocabulary/hasPropertyUri": [
+        {
+          "@id": "http://sinopia.io/vocabulary/hasValidationDataType"
```

Comment:
should this be "http://sinopia.io/vocabulary/propertyAttribute/hasValidationDataType"  (or even "literalAttribute")